modules/text_on_image/text_on_image_NUM_YOUR_NAME_.html

```diff
@@ -0,0 +1,63 @@
+<div class="content">
+    <p class="white-text small-text">para1</p>
+    <p class="white-text medium-text">para2</p>
+    <p class="white-text large-text">para3</p>
+</div>
+<!-- change the path for your use-->
+<img src="path1" alt="" class="top-left-image"> 
+<img src="path2" alt="" class="image-right-bottom">
+
+<style>
+    body {

```

Comment:
you can avoid giving the entire body some styles because that may affect the entire website. You can instead: create a div and give it  a width of 100% so that it fits any parent tag. This is not mandatory though but just makes your module high-class.